Mathlib/Order/Minimal.lean

```diff
@@ -52,24 +52,6 @@ section LE
 
```

Comment:
The file-level docstring is not correct any more, as it says this file defines minimality. Can you update it?